lib/ruby-mpd/plugins/controls.rb

```diff
@@ -21,6 +21,7 @@ def pause=(toggle)
       def play(pos = nil)
         if pos.is_a?(Hash)
           if pos[:id]
+            # TODO What is priority?
             send_command :playid, priority, pos[:id]
           else
             raise ArgumentError, 'Only :id key is allowed!'
```

Comment:
OK, I did a bit of test refactoring (see #b3ee1b1) to make further testing easier. I now have 1 failing test though. In the ```play``` method in ```lib/ruby-mpd/plugins/controls.rb``` it calls **priority** which does not appear to be defined anywhere? This is highlighted when I try to mock it in the tests. Should priority be an argument?